Cargo.toml

```diff
@@ -80,7 +80,7 @@ percent-encoding = "2.1"
 pin-project = "1.0"
 quick-xml = { version = "0.23", features = ["serialize"] }
 radix_trie = { version = "0.2", optional = true }
-reqsign = "0.4"
+reqsign = { git = "https://github.com/Xuanwo/reqsign", branch = "main" }
```

Comment:
`reqsign` needs a new release?

Reply:
Yes, reqsign needs a release. But it okay for this PR to merge as is. I will handle other releasing parts.